docs/architecture/README.md

```diff
@@ -0,0 +1,59 @@
+<!--
+order: 1
+parent:
+  title: Architecture Decision Records (ADR)
+  order: 10
+-->
+
+# Architecture Decision Records (ADR)
+
+This is a location to record all high-level architecture decisions for new feature and module proposals in the Cosmos Hub.
+
+An Architectural Decision (**AD**) is a software design choice that addresses a functional or non-functional requirement that is architecturally significant.
+An Architecturally Significant Requirement (**ASR**) is a requirement that has a measurable effect on a software system’s architecture and quality.
+An Architectural Decision Record (**ADR**) captures a single AD, such as often done when writing personal notes or meeting minutes; the collection of ADRs created and maintained in a project constitute its decision log. All these are within the topic of Architectural Knowledge Management (AKM).
+
+You can read more about the ADR concept in this [blog post](https://product.reverb.com/documenting-architecture-decisions-the-reverb-way-a3563bb24bd0#.78xhdix6t).
```

Comment:
Maybe a better link, from which some of this text seems to be copied from https://adr.github.io/